spec/models/mixins/ownership_mixin_spec.rb

```diff
@@ -0,0 +1,26 @@
+describe "Ownership" do
+  describe "reporting" do
+    describe "searching by current ldap group" do
+      let(:exp)    { {"="=>{"field"=>"Vm-owned_by_current_ldap_group", "value"=>"true"}} }
+      let(:report) { MiqReport.new.tap {|r| r.db = 'Vm'} }
+      let(:search_opts) { { :filter => MiqExpression.new(exp), :per_page => 20, :named_scope => :with_miq_group } }
+
+      before do
+        @miq_group = FactoryGirl.create :miq_group
+        User.current_user = FactoryGirl.create :user,
+                                               :current_group => @miq_group,
+                                               :miq_groups => [@miq_group]
+        expect(User).to receive(:server_timezone).and_return("UTC")
+
+        vm_with_group = FactoryGirl.create :vm_vmware, :miq_group_id => @miq_group.id
+        vm_no_group   = FactoryGirl.create :vm_vmware
+      end
+
+      it "returns results only part of the miq_group" do
+        vm_ids = report.paged_view_search(search_opts).first.map(&:id)
+        vms    = Vm.where(:id => vm_ids)
+        expect(vms.all? {|vm| vm.miq_group_id == @miq_group.id }).to be true
```

Comment:
Yeah, wasn't sure if we were starting from a clean state (no records in the `vms` table), so I was mostly working around that by making sure the items returned only are part of the `MiqGroup`, and not ones created without.

This is silly now that I think about it because I am making the `MiqGroup` as part of this test, so only the `vms` that I am creating here should be include... I will make that change.

For the record, I think this test is kind of a hack in it of itself, but I was mostly doing it as a way of confirming existing functionality, and that I didn't break it when adding the new additions (which technically I did because `with_miq_group` is now a required scope).